enyojs/ares-project
# ENYO-1741: A Sign-In error Pop-up appears if the user open the "Project ...

...Properties" Pop-up and he is not authentified in his Phonegap account.

The Sign-In error Pop-up (Ares.SignInErrorPopup) is inherited from the" Ares.Error" Pop-up, and have in addition a "Sign In" button that show the "Account" Pop-up.

How to test : 
1 - Enter an incorrect id/password for the Phonegap Account authentication in the Pop-up "Account".
2- Open the project properties Pop-up
3 - An error Pop-up should appear, the error Pop-up has the "Sign In" button
4 - By tapping on the sign in button, the "Account" Pop-up should appear to let the user re-enter his loggin and password.

Ready for review

Tested on Chrome (Windows 7)
Enyo-DCO-1.1-Signed-off-by: Rafik Adiche mahmoud-rafik.adiche@hp.com
